src/app/components/dataset-details/dataset-details.component.ts

```diff
@@ -70,4 +72,12 @@ export class ItemDetailsComponent {
   isNotEmtpy() {
     return !!this.datasetDetails && Object.keys(this.datasetDetails).length > 0
   }
+
+  getDistributions(): any[] {
+    let distributions = this.datasetDetails["resources"];
```

Comment:
Tiny suggestions: It is a good practice using const as much as possible, if the variable will not be/does not need to be reassigned. It reduces changes of side effects, and antecipates coding issues.

Reply:
good point